sashimi/hardware/lasers/__init__.py

```diff
@@ -0,0 +1,54 @@
+from enum import Enum
+
+
+class LaserState(Enum):
+    ON = 1
+    OFF = 2
+
+
+class LaserException(Exception):
+    pass
+
+
+class LaserWarning(Warning):
+    pass
+
+
+class AbstractLaser:
```

Comment:
An optional suggetion: call it AbstractLightSource (a small change, but then we can claim sashimi supports epiflourescence microscopes)